dashboard/app/models/pd/workshop.rb

```diff
@@ -183,6 +183,10 @@ def self.in_state(state)
     end
   end
 
+  def course_name
+    course == Pd::Workshop::COURSE_ADMIN ? 'Administrator' : course
+  end
+
```

Comment:
So far this is US-only and not translated, but that's a good point. I like this better. 

Ideally we'd have a similar thing for subject, and this name would be returned in the API (to be displayed in the UI) instead of the DB value.